.github/workflows/ci_tests.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         if: github.event_name == 'pull_request' && contains(matrix.module, ':firebase-firestore')
         run: |
           cd firebase-firestore
-          terraform apply -var-file=../google-services.json -auto-approve
+          terraform apply -var-file=../google-services.json -auto-approve &> /dev/null
```

Comment:
This effectively silences the output of the command. Is it intentional?

Reply:
Yes, the terraform apply kept repeating the "index already created" error message, and what we can do to eliminate the error message is very limited. Silencing it for not to avoid alerting everyone who saw the message.

Reply:
but what if `terraform apply` fails for _another_ legitimate reason? then we won't be able to tell the failure cause because it will be missing from the logs.

Reply:
Terraform is used for index creation right at the moment. If there are changes like project/permission change that eventually leads to the failure in index creation, this will be reflected on the test failure (index missing). Then we need to trace back where index were created and remove the `&> /dev/null`.